students/mark_luckeroth/lesson04/kata.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+import random
+
+def make_words(filename):
+    f = open(filename, 'r')
+    lines = f.readlines()
+    header_size = 0
+    for ind, line in enumerate(lines):
+        if line.startswith("*** START OF"):
+            header_size = ind
+        if line.startswith("End of the Project Gutenberg"):
+            footer_ind = ind
+            break
+    data = lines[header_size+1:footer_ind]
+    text = " ".join(data)
+    f.close()
+    return text.split()
+
+def build_trigrams(words, max=10000):
+    """
+    build up the trigrams dict from the list of words
+
+    returns a dict with:
+       keys: word pairs
+       values: list of followers
+    """
+    count = min(len(words)-2, max)
+    trigrams = {}
+    for i in range(count):
+        pair = (words[i], words[i+1])
+        follower = words[i+2]
+        if pair in trigrams.keys():
+            trigrams[pair].append(follower)
+        else:
+            trigrams[pair] = [follower]
+    return trigrams
+
+def build_text(trigram, max=200, seed=['I',  'wish']):
```

Comment:
best not to use mutable objects as default keyword values.

https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments